### PR TITLE
Trapper packs and grass headwear

### PR DESCRIPTION
**Pull request by Sathra225**
#### Summary
Content "More primitive backpacks and grass keffiyeh"

#### Purpose of change
Adds 2 new backpacks made of cloth and wood, based on the Nelson Trapper packs for early innawoods games. Also adds a grass sheet version of the keffiyeh, as early head warmth/mouth was missing.

#### Describe the solution
Adds the items and crafting recipes.

#### Describe alternatives you've considered
None

#### Testing
Made changes, loaded save. Made and wore both packs and the keffiyeh. No errors.

#### Additional context
EDIT: Shifted to mainline as requested. Game still loads, but not confident enough with spawn locations to add them.

### CODE REVIEW

**Comment by Sathra225** [comment]
@Light-Wave for review

**Comment by TheShadowFerret** [comment]
Can vanilla have these too? ❤️

**Comment by NetSysFire** [comment]
I do not see why not. Only thing I would look into is the actual realism of the grass keffiyeh.

**Comment by Sathra225** [comment]
I'm not sure how feasible these would actually be for mainline, as the small one is 5kg and the big one is 10kg. They're big, crude packs. In mainline it would make more sense to to have them made of aluminium replacing the wood if player-craftable.

As for the grass keffiyeh...its more of a best fit for the concept (which is basically a loose grass sheet, secured to the head that you can tuck into itself to cover the mouth and nose). So its not quite a headwrap, its more loose. 
Are there real examples that I know of? Doubtful, it'd be uncomfortable and archeological records wouldn't show it as its basically a grass sheet and some cords. 
Is it something a survivor would do if they had nothing else? Yes, kinda. It'd be more likely to cover the head with a worn grass sheet and hunch, but the current sheets aren't worn over the head for some reason and fixing that would involve adding flags to something I'm not actually sure of the size of.

**Comment by Light-Wave** [comment]
I agree that these fits beter in mainline than in Innawood. Putting it in mainline also means more scrutiny regarding how balanced these things are, which I would also consider a good thing.

**Review by Light-Wave — CHANGES_REQUESTED** [review]
Request these changes be moved to mainline.

**Comment by Sathra225** [comment]
Shifted over to main.

File: data/json/items/armor/storage.json [hunk 1]
```diff
@@ -2591,5 +2591,64 @@
     ],
     "material_thickness": 2,
     "armor": [ { "encumbrance": [ 3, 80 ], "coverage": 10, "covers": [ "torso" ], "specifically_covers": [ "torso_hanging_back" ] } ]
+  },
+  {
+    "id": "bigtrapper_pack",
+    "type": "ARMOR",
+    "name": { "str": "big trapper pack" },
+    "description": "A wooden frame with waterproofed cloth stretched around it, and a drawstring at the top to hold things in.  It has straps to wear it on your back",
```
Comment (casswedson) [inline data/json/items/armor/storage.json:2599]:
```suggestion
    "description": "A wooden frame with waterproofed cloth stretched around it, and a drawstring at the top to hold things in.  It has straps to wear it on your back.",
```

**Pull request merged**